### ISSUE TITLE
Triplet loss finetune

### ISSUE THREAD

**Issue by tangchen2** [davidsandberg/facenet#1075]
Hi, when i use FaceNet train CASIA-Webface, i first use Softmax Loss, and after 90 epoch, the model gets 98% accuracy and 95% validation rate.
Then, i want to use Triplet Loss to finetune the model, however, i can't find suitable parameters to improve the model.
I have try the learning rate from 0.001 to 1e-7, the model performance all decreased (especially the validate rate)
and I am really confused about that, how do you guys finetune the model with Triplet Loss?

**Comment by xlphs** [comment]
I have written some comments in this thread to help. https://github.com/davidsandberg/facenet/issues/1012#issuecomment-490119259
